sdk/resourcemanager/Azure.ResourceManager.Core/OLD README.md

```diff
@@ -0,0 +1,77 @@
+# Azure ResourceManager Core client library for .NET
+
+This package follows the [new Azure SDK guidelines](https://azure.github.io/azure-sdk/general_introduction.html) which provide a number of core capabilities that are shared amongst all Azure SDKs, including the intuitive Azure Identity library, an HTTP Pipeline with custom policies, error-handling, distributed tracing, and much more.
+
+## Getting started 
+
+### Install the package
+
+Install the Azure Resources management core library for .NET with [NuGet](https://www.nuget.org/):
+
+```PowerShell
+Install-Package Azure.ResourceManager.Core -Version 1.0.0-beta.1
+```
+
+### Prerequisites
+
+* You must have an [Azure subscription](https://azure.microsoft.com/free/)
+
+### Authenticate the Client
+
+To create an authenticated client and start interacting with Azure resources, please see the [quickstart guide here](https://github.com/Azure/azure-sdk-for-net/blob/master/doc/mgmt_preview_quickstart.md)
```

Comment:
let us use this one:
https://aka.ms/azsdk/dotnet/mgmt

in case future changes